src/htmx.js

```diff
@@ -289,27 +288,26 @@ return (function () {
                 // @ts-ignore type mismatch between DocumentFragment and Element.
                 // TODO: Are these close enough for htmx to use interchangeably?
                 return documentFragment.querySelector('template').content;
-            } else {
-                var startTag = getStartTag(resp);
-                switch (startTag) {
-                    case "thead":
-                    case "tbody":
-                    case "tfoot":
-                    case "colgroup":
-                    case "caption":
-                        return parseHTML("<table>" + resp + "</table>", 1);
-                    case "col":
-                        return parseHTML("<table><colgroup>" + resp + "</colgroup></table>", 2);
-                    case "tr":
-                        return parseHTML("<table><tbody>" + resp + "</tbody></table>", 2);
-                    case "td":
-                    case "th":
-                        return parseHTML("<table><tbody><tr>" + resp + "</tr></tbody></table>", 3);
-                    case "script":
-                        return parseHTML("<div>" + resp + "</div>", 1);
-                    default:
-                        return parseHTML(resp, 0);
-                }
+            }
+            var startTag = getStartTag(resp);
+            switch (startTag) {
+                case "thead":
+                case "tbody":
+                case "tfoot":
+                case "colgroup":
+                case "caption":
+                    return parseHTML("<table>" + resp + "</table>", 1);
```

Comment:
I see `resp` is a string, so can we just use template literals here and all the similar places afterwards?

```suggestion
                    return parseHTML(`<table> ${resp}</table>`, 1);
```

Reply:
I think they are keeping the IE11 compatibility for nostalgic reasons alive, and this would break it. :)

Reply:
I also prefer template literals. In that case I didn't change the original code, as this is subjective.
If it's wished, I can change it. There are probably many more candidates for template literals in the code.

Reply:
Thanks for bringing it to my attention @andryyy , that seems like a valid point. I guess we can never get rid of the backward compatibility issue till IE(n) is alive 😀